versions/3.0.md

```diff
@@ -1184,6 +1185,23 @@ Locating a linked resource may be performed by either a `href` or `operationId`.
 In the case of an `operationId`, it must be unique and resolved in the scope of the OAS document.
 Because of the potential for name clashes, consider the `href` syntax as the preferred method for specifications with external references.
 
+When the `url` attribute is provided, the target host may be overridden by the value provided.  For example:
+
+```
+components:
+  links:
+    UserRepositories:
+      operationId: getRepositoriesByOwner
+      url: $responseHeaders.link
```

Comment:
Yes, this is a minor doc change, we'll address that while starting a sweep of the docs next week